docs/index.md

```diff
@@ -33,7 +33,7 @@ What is HTCondor-CE?
 --------------------
 
 HTCondor-CE is a special configuration of the HTCondor software designed as a Compute Entrypoint.
-It is configured to use the HTCondor [Job Router daemon](https://htcondor.readthedocs.io/en/stable/grid-computing/job-router.html)
+It is configured to use the HTCondor [Job Router daemon](https://htcondor.readthedocs.io/en/latest/grid-computing/job-router.html)
```

Comment:
```suggestion
It is configured to use the HTCondor [Job Router daemon](https://htcondor.readthedocs.io/en/v9_0/grid-computing/job-router.html)
```

Since the latest HTCondor-CE version docs point to the HTCondor 9.0 manual, let's do the same thing here